tfgrid3/umbrel/dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM debian:stable-slim
+
+ENV DEBIAN_FRONTEND=noninteractive \
+    UMBREL_VERSION="release" \
+    UMBREL_REPO="getumbrel/umbrel" \
+    UMBREL_INSTALL_PATH="/umbrel" 
+
+RUN apt update && \
+    apt -y install wget curl vim net-tools iputils-ping openssh-server docker.io \
+    fswatch jq rsync sudo iproute2 git gettext-base python3 gnupg avahi-daemon avahi-discover libnss-mdns nginx ufw
```

Comment:
it is recommended to clean up the apt cache by removing `/var/lib/apt/lists` which reduces the image size since the apt cache isn’t stored in a layer.
you can achieve this by appending `&& rm -rf /var/lib/apt/lists/*` to the same RUN statement where you install the pkgs.
see here for more info:
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run